vip-plugins/vip-plugins.php

```diff
@@ -222,7 +224,11 @@ function wpcom_vip_site_option_active_sitewide_plugins( $value, $option ) {
 function wpcom_vip_pre_update_option_active_plugins( $value, $old_value, $option ) {
 	$code_plugins = wpcom_vip_get_filtered_loaded_plugins();
 
-	if ( ! empty( $value ) && ! empty( $code_plugins ) ) {
+	if ( empty( $value ) ) {
+		$value = array();
+	}
+
+	if ( ! empty( $code_plugins ) ) {
```

Comment:
Same as above. If we feel like the check helps the code to perform less work, it should be used even in `wpcom_vip_option_active_plugins`. I personally would omit this conditional, as it feels being extra.